t/NobelPrize.t

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use Test::More;
+use DDG::Test::Spice;
+
+spice is_cached => 1;
+
+ddg_spice_test(
+    [ 
+        "DDG::Spice::NobelPrize" 
+    ],
+    
+    # Both year and category
+    'nobel 1950 physics' => test_spice(
+        '/js/spice/nobel_prize/1950-physics',
+        call_type => 'include',
+        caller => 'DDG::Spice::NobelPrize'
+    ),
+    
+    # Only year
+    'winner 2000 nobel' => test_spice(
+        '/js/spice/nobel_prize/2000-',
+        call_type => 'include',
+        caller => 'DDG::Spice::NobelPrize'
+    ),
+    
+    # Only Category is given
+    'nobel winners in medicine' => test_spice(
+        '/js/spice/nobel_prize/-medicine',
+        call_type => 'include',
+        caller => 'DDG::Spice::NobelPrize'
+    ),
+    
+    # Take only first of given years
+    'nobel laureates of 2004 and 2005' => test_spice(
```

Comment:
This shouldn't show a result. We're not correctly answering the question asked.

If we receive the query "height of tom cruise and tom hanks" it would be incorrect to just show the result for one of them. I see handling this query in the same way.